keras_core/ops/image.py

```diff
@@ -55,7 +55,35 @@ def compute_output_spec(self, image):
 def resize(
     image, size, method="bilinear", antialias=False, data_format="channels_last"
 ):
-    # TODO: add docstring
+    """Resize images to size using the specified method.
+
+    Args:
+        image: Input image or batch of images. Must be 3D or 4D.
+        size: Size of output image in (height, width) format.
+        method: Interpolation method. Defaults to 'bilinear'.
```

Comment:
List available methods. They're code keywords, so use both backticks and quotes, e.g. `"bilinear"`